bundles/org.openhab.voice.watsonstt/src/main/resources/OH-INF/i18n/watsonstt_it.properties

```diff
@@ -6,8 +6,8 @@ voice.config.watsonstt.group.authentication.label = Autenticazione
 voice.config.watsonstt.group.authentication.description = Informazioni per la connessione all'istanza Watson Speech-to-Text.
 voice.config.watsonstt.group.stt.label = Configurazione STT
 voice.config.watsonstt.group.stt.description = Parametri per l'API Watson Speech-to-Text.
-voice.config.watsonstt.inactivityTimeout.label = Timeout Inattività
-voice.config.watsonstt.inactivityTimeout.description = Il tempo in secondi dopo il quale, se viene rilevato solo silenzio (nessun discorso) nell'audio, la connessione viene chiusa.
+voice.config.watsonstt.maxSilenceSeconds.label = Timeout Inattività
```

Comment:
To be removed. 
To be done later in Crowdin.

Reply:
So, should I avoid modify property files other than the English one?

Reply:
Yes